sublime-package.json

```diff
@@ -0,0 +1,738 @@
+                      "description": "Enables unexpected token diagnostics.",
+                      "scope": "window"
+                    },
+                    "intelephense.diagnostics.duplicateSymbols": {
+                      "type": "boolean",
+                      "default": true,
+                      "description": "Enables duplicate symbol diagnostics.",
+                      "scope": "window"
+                    },
+                    "intelephense.diagnostics.argumentCount": {
+                      "type": "boolean",
+                      "default": true,
+                      "description": "Enables argument count diagnostics.",
+                      "scope": "window"
+                    },
+                    "intelephense.diagnostics.typeErrors": {
+                      "type": "boolean",
+                      "default": true,
+                      "description": "Enables diagnostics on type compatibility of arguments, property assignments, and return statements where types have been declared.",
+                      "scope": "window"
+                    },
+                    "intelephense.diagnostics.deprecated": {
+                      "type": "boolean",
+                      "default": true,
+                      "description": "Enables deprecated diagnostics.",
+                      "scope": "window"
+                    },
+                    "intelephense.diagnostics.languageConstraints": {
+                      "type": "boolean",
+                      "default": true,
+                      "description": "Enables reporting of various language constraint errors.",
+                      "scope": "window"
+                    },
+                    "intelephense.diagnostics.implementationErrors": {
+                      "type": "boolean",
+                      "default": true,
+                      "description": "Enables reporting of problems associated with method and class implementations. For example, unimplemented methods or method signature incompatibilities.",
+                      "scope": "window"
+                    },
+                    "intelephense.runtime": {
+                      "type": "string",
+                      "description": "Path to a Node.js executable. Use this if you wish to use a different version of Node.js. Defaults to Node.js shipped with VSCode.",
+                      "scope": "machine"
+                    },
+                    "intelephense.maxMemory": {
+                      "type": "number",
+                      "description": "Maximum memory (in MB) that the server should use. On some systems this may only have effect when runtime has been set. Minimum 256.",
+                      "scope": "window"
+                    },
+                    "intelephense.licenceKey": {
+                      "type": "string",
+                      "description": "DEPRECATED. Don't use this. Go to command palette and search for enter licence key.",
```

Comment:
Description not relevant?

Reply:
https://github.com/sublimelsp/LSP-intelephense/pull/48/commits/d263131c8e68464da3e352792a695250053d7f6a#diff-1b30ef1dc3d227d786b769b6c3fff3c23598512f28d3f2f0379e8c542d4c20a6R505-R508 :+1: